gnomad_constraint/pipeline/constraint_pipeline.py

```diff
@@ -180,7 +193,10 @@ def main(args):
 
                 # Filter input Table and add annotations used in constraint
                 # calculations.
-                ht = prepare_ht_for_constraint_calculations(ht)
+                ht = prepare_ht_for_constraint_calculations(
+                    ht, require_exome_coverage=False if data_type != "exomes" else True
```

Comment:
Sorry, one more change, this just reads easier
```suggestion
                    ht, require_exome_coverage=True if data_type == "exomes" else False
```